plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -139,6 +142,13 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         };
     }
 
+    @Test
+    @Override
+    public void testUpdateRowConcurrently()
+    {
+        abort("Postgresql doesn't support concurrent update of different columns in a row");
```

Comment:
How does it fail then?

Reply:
There will be assertion error, the expected row not matches.
e.g, there are two columns: c1, c2 in the row, starts: `[0,0]`, update them +1 separately, expected : `[1, 1]`, but current implement would override the whole row in a update, it's possible result in `[0,1]` or `[1, 0]` if concurrent update.